TestOnly/pom.xml

```diff
@@ -118,7 +118,7 @@
     <dependency>
        <groupId>org.codehaus.jackson</groupId>
        <artifactId>jackson-mapper-asl</artifactId>
-       <version>1.9.9</version>
+       <version>1.9.13</version>
```

Comment:
Isn't the warning about versions <**=**1.9.13 ?

Reply:
removed `jackson-mapper-asl` and rewrote tests.